client/ayon_deadline/lib.py

```diff
@@ -8,3 +10,33 @@
     "karma_rop", "vray_rop", "redshift_rop",
     "renderFarm", "usdrender", "publish.hou"
 ]
+
+
+def get_ayon_render_job_envs() -> "dict[str, str]":
+    """Get required env vars for valid render job submission."""
+    return {
+        "AYON_LOG_NO_COLORS": "1",
+        "AYON_RENDER_JOB": "1",
+        "AYON_BUNDLE_NAME": os.environ["AYON_BUNDLE_NAME"]
+    }
+
+
+def get_instance_job_envs(instance) -> "dict[str, str]":
+    """Add all job environments as specified on the instance and context.
+
+    Any instance `job_env` vars will override the context `job_env` vars.
+    """
+    key = "job_env"
```

Comment:
`job_env` is too generic not following pyblish key convention (camelcase), I propose `farmJobEnv`?

But as mentioned here https://github.com/ynput/ayon-deadline/pull/32#issuecomment-2399394081 this does not allow to differentiate between render and publish jobs.

Reply:
> But as mentioned here https://github.com/ynput/ayon-deadline/pull/32#issuecomment-2399394081 this does not allow to differentiate between render and publish jobs.

Correct, but that _can_ be a separate issue/PR - I have some ideas; but I'd rather keep it out of this one for now. I'd like to overhaul some other stuff with that to clean things up further anyway.

Reply:
> job_env is too generic not following pyblish key convention (camelcase), I propose farmJobEnv?

Fixed with https://github.com/ynput/ayon-deadline/pull/32/commits/0bfa3e192d85ecf68e5abf321c355d7fcf1fff7e